### PR TITLE
HW02

### PR DESCRIPTION
**Pull request by heyeugene**

### CODE REVIEW

File: HW_02/Eugene_Anikin/eugene_anikin_hw_02_t_02.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Lorem ipsum
+puts 'Try just add 1, 3, 4, 6'
+x = 1 / 3 + 4 * 6
+puts "x is: #{x}"
```
Comment (labs-ruby) [inline HW_02/Eugene_Anikin/eugene_anikin_hw_02_t_02.rb]: why 1 / 3 = 0?

Comment (heyeugene) [reply HW_02/Eugene_Anikin/eugene_anikin_hw_02_t_02.rb]: when two integers are divided, the result is also an integer

**Comment by alexshgov** [comment]
WTF, I don't understand your commit messages!

**Comment by alexshgov** [comment]
Delete what? Add what?

**Pull request merged**